libraries/src/HTML/Helpers/Icons.php

```diff
@@ -21,6 +22,45 @@
  */
 abstract class Icons
 {
+	/**
+	 * Method to write a `<span>` element for an icon
+	 *
+	 * @param   string  $icon     The functional name for an icon.
+	 * @param   string  $srOnly   Screen Reader text if no visible text is placed
+	 * @param   array   $attribs  Attributes to be added to the wrapping element
+	 *
+	 * @return  string
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public static function icon(string $icon = '', string $srOnly = '', array $attribs = [])
```

Comment:
Icon should not be optional.

```suggestion
	public static function icon(string $icon, string $srOnly = '', array $attribs = [])
```